.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
       env: JEKYLL_VERSION=3.7.0
     - rvm: *latest_ruby
       env: JEKYLL_VERSION=4.0.0.pre.alpha1
+    - rvm: *latest_ruby
+      env: FARADAY_VERSION=0.17.3
+    - rvm: *latest_ruby
+      env: FARADAY_VERSION=1.0.0
```

Comment:
Let us relax these constraints a bit since the `Gemfile` uses the `~>` operator. 
`~> 0.17.3` translates into `">= 0.17.3", "< 0.18.0"` and `~> 1.0.0` translates into `">= 1.0.0", "< 1.1.0"`

Instead, having `~> 0.17` and `~> 1.0` will be future-proof and shall always test with the latest available version in the `v0.x` and `v1.x` series..

Reply:
On further thought, I feel this should be clubbed with Jekyll versions..:
```yaml
    - rvm: *latest_ruby
      env: JEKYLL_VERSION=3.7 FARADAY_VERSION=0.17
    - rvm: *latest_ruby
      env: JEKYLL_VERSION=3.7 FARADAY_VERSION=1.0
    - rvm: *latest_ruby
      env: JEKYLL_VERSION=4.0 FARADAY_VERSION=0.17
    - rvm: *latest_ruby
      env: JEKYLL_VERSION=4.0 FARADAY_VERSION=1.0
```
And also have similar set up for AppVeyor builds as well.

Reply:
@ashmaroli thanks! I did it.